cmd/kubeapps-apis/buf.yaml

```diff
@@ -0,0 +1,13 @@
+version: v1beta1
+name: "buf.build/kubeapps/kubeapps"
+build:
+  roots:
+    - proto
+lint:
+  use:
+    - DEFAULT
+  except:
+    - RPC_REQUEST_RESPONSE_UNIQUE
```

Comment:
Yes, because we're defining the request and responses in our core proto definition so that they can be used in all plugin implementations, without *forcing* the plugin to use them (at least, initially,) as there may be unforeseen exceptions.